<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectWindow</class>
 <widget class="QMainWindow" name="CreateProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Audio Project </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="projectNameLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>Project Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="audioFormatBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E1E2F;
color: white;
</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>MP3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WAV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FLAC</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1E1E2F;
    border-radius: 12px;
    color: white;
    padding: 10px;  
    font-size: 12px;
}
QPushButton:hover {
    background-color: #3A3A5C;
}</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="projectNameInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E1E2F;
color: white;
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="durationInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E1E2F;
color: white;
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="samplingFrequencyBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E1E2F;
color: white;
</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>44.1 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88.2 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192 kHz</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="samplingFrequencyLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Sampling Frequency (Rate)</string>
    </property>
   </widget>
   <widget class="QLabel" name="audioFormatLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Audio Format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1E1E2F;
    border-radius: 12px;
    color: white;
    padding: 10px;  
    font-size: 12px;
}
QPushButton:hover {
    background-color: #3A3A5C;
}</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="durationLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Duration (seconds)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1E1E2F;
    border-radius: 12px;
    color: white;
    padding: 10px;  
    font-size: 12px;
}
QPushButton:hover {
    background-color: #3A3A5C;
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
